README.rst

```diff
@@ -435,6 +426,25 @@ The recipe supports the following options:
     Allows you to specify directories and/or files which you don't want to be
     checked. Default is none.
 
+**isort**
+    If set to True, use `isort`_ to scan Python files and report about imports
+    not sorted according to the **isort-options**. Default is ``False``.
+
+**isort-exclude**
+    Allows you to specify directories and/or files which you don't want to be
+    checked. Default is none.
+
+**isort-[option]**
+    Allows you to change the way imports should be sorted. Default is to follow
+    `plone.api conventions`_. See `isort documentation for available options
+    <http://pypi.python.org/pypi/isort>`_::
+
+        ...
+        isort = True
+        isort-force-alphabetical-sort = True
+        isort-force_single_line_imports = True
+        isort-diff = True
+
```

Comment:
@saily what's the benefit of this compared to just passing the command line options in a single variable as my original code was doing?

That's just so much verbose and on top of that you can not play with isort on the commandline, check if you like the result and then **verbatim** copy that to your buildout.cfg and you are good to go.

With this approach you have to add the ``isort-`` prefix, the `` = True`` and so on...